### DISCUSSION TITLE
Default User & /canvas/register Route

### DISCUSSION THREAD

**Discussion by james0r** [austintoddj/canvas#1058] [General]
Not seeing any mention in the docs about what the default user/pass is. Also my /canvas/register route just redirects to /canvas/login. Is this expected default behavior?

Thanks!

**Comment by austintoddj** [comment]
The default user is coming [from this line](https://github.com/austintoddj/canvas/blob/master/src/Console/InstallCommand.php#L71). There's actually no register route in the app at all. With the right permissions (Admin), you can create users in the app. But there was never the intention of having outside users sign up and access the blog you're hosting.

**Comment by austintoddj** [comment]
Closing this out with [Canvas v7.0.0](https://github.com/austintoddj/canvas/releases/tag/v7.0.0).

There is no Canvas-owned register/login and no default Canvas password. Create users in the host app, then:

```bash
php artisan canvas:make-admin you@example.com
```

Visit /canvas while signed in as that user.
